wiki/Storyboard_Scripting/en.md

```diff
@@ -1,17 +1,20 @@
 # Storyboard scripting
 
-![An example of scripting in .osb.](img/SBS_Base.jpg "An example of scripting in .osb.")
+![An example of scripting in .osb](img/SBS_Base.jpg "An example of scripting in .osb")
```

Comment:
Maybe update the image with windows 10 ui.

Reply:
I'd love to do that, but in a separate PR (dunno how soon I can get ahold of my PC)